modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/I18n/Language.json

```diff
@@ -390,6 +393,7 @@
 	"the-ip-must-not-have-spaces": "The IP must not have spaces.",
 	"the-mac-must-not-have-spaces": "The MAC must not have spaces.",
 	"the-project's-support-level-is-displayed-here-for-projects-with-ticketing-support": "The project's Support Level is displayed here for projects with ticketing support.",
+	"the-requested-activation-key-exceeds-the-purchased-subscriptions-for-this-liferay-project.-in-case-of-unauthorized-use,-liferay-can-request-financial-compensation-for-breach-of-use.": "The requested activation key exceeds the purchased subscriptions for this Liferay project. In case of unauthorized use, Liferay can request financial compensation for breach of use. For more information, please reach out to your Liferay contact or sales@liferay.com.",
```

Comment:
No dots and comma

```suggestion
	"the-requested-activation-key-exceeds-the-purchased-subscriptions-for-this-liferay-project-in-case-of-unauthorized-use-liferay-can-request-financial-compensation-for-breach-of-use": "The requested activation key exceeds the purchased subscriptions for this Liferay project. In case of unauthorized use, Liferay can request financial compensation for breach of use. For more information, please reach out to your Liferay contact or sales@liferay.com.",
```